kotlinx-coroutines-core/build.gradle.kts

```diff
@@ -306,3 +306,7 @@ artifacts {
 tasks.named("dokkaHtmlPartial") {
     dependsOn(jvmJar)
 }
+
+atomicfu {
+    transformJs = false
+}
```

Comment:
Can't this be put into `buildSrc/src/main/kotlin/configure-compilation-conventions.gradle.kts`? The `atomicfu` plugin is applied there, and it would be nice to keep the related logic close together.

Reply:
I applied this config in `configure-compilation-conventions` in the last commit)